src/Microsoft.ServiceFabric.Actors.KVSToRCMigration/MigrationOrchestratorBase.cs

```diff
@@ -91,13 +91,16 @@ public ICommunicationListener GetMigrationCommunicationListener()
 
                     var webHostBuilder =
                         new WebHostBuilder()
-                            .UseKestrel()
+                            .UseServiceFabricIntegration(listener, ServiceFabricIntegrationOptions.UseUniqueServiceUrl)
+                            .UseHttpSys(options =>
+                            {
+                                options.UrlPrefixes.Add(listener.GetListenerUrl());
+                            })
                             .ConfigureServices(
                                 services => services
                                     .AddSingleton<IMigrationOrchestrator>(this))
-                            .UseServiceFabricIntegration(listener, ServiceFabricIntegrationOptions.UseUniqueServiceUrl)
                             .UseStartup<Startup>()
-                            .UseUrls(url)
+                            .UseUrls(listener.GetListenerUrl())
```

Comment:
This is redundant as the url has been specified in Options.UrlPrefixes. Can be removed.

Reply:
Fixed